fvm/bootstrap.go

```diff
@@ -77,6 +77,7 @@ type BootstrapParams struct {
 	storagePerFlow                   cadence.UFix64
 	restrictedAccountCreationEnabled cadence.Bool
 	setupEVMEnabled                  cadence.Bool
+	evmAbiOnly                       cadence.Bool
```

Comment:
Given this is just a feature flag and the value is only used in Go / not passed to Cadence, this can just be a plain `bool` (and usages below as well)
```suggestion
	evmAbiOnly                       bool
```

Reply:
Yeah, I was wondering about it as well, but it seems that every config is a Cadence value. Not sure why.